x-pack/plugins/siem/public/components/source_destination/source_destination_arrows.tsx

```diff
@@ -214,8 +214,8 @@ export const SourceDestinationArrows = pure<{
   const maybeSourceBytesPercent =
     maybeSourceBytes != null && maybeDestinationBytes != null
       ? getPercent({
-          numerator: Math.min(Number(maybeSourceBytes), Number(maybeDestinationBytes)),
-          denominator: Math.max(Number(maybeSourceBytes), Number(maybeDestinationBytes)),
+          numerator: Number(maybeSourceBytes),
+          denominator: Number(maybeSourceBytes) + Number(maybeDestinationBytes),
```

Comment:
Since there were no test coverage of this and this turned out to be a bad data bug, I would write a few unit tests which exercise the bug and proves that your fix does indeed fix it and then we avoid any further regressions.

Reply:
Done